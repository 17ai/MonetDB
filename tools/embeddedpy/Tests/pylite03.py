# monetdblite.create statement with multiple columns

import monetdblite, shutil, os
dbfarm = '/tmp/pylite_dbfarm'
if os.path.isdir(dbfarm): shutil.rmtree(dbfarm)
monetdblite.init(dbfarm)

import numpy
monetdblite.create('pylite03', ['i', 'j', 'k', 'l', 'm'], numpy.arange(100000).reshape((5,20000)))
res = monetdblite.sql('select * from pylite03')
print res
print 'Count', len(res['i'])

shutil.rmtree(dbfarm)

