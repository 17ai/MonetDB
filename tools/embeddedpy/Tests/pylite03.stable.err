stderr of test 'pylite03` in directory 'tools/embeddedpy` itself:


# 14:17:06 >  
# 14:17:06 >  "/usr/bin/python2" "pylite03.py" "pylite03"
# 14:17:06 >  


# 14:17:07 >  
# 14:17:07 >  "Done."
# 14:17:07 >  

