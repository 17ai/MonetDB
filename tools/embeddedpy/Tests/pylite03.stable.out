stdout of test 'pylite03` in directory 'tools/embeddedpy` itself:


# 14:17:06 >  
# 14:17:06 >  "/usr/bin/python2" "pylite03.py" "pylite03"
# 14:17:06 >  

Making directory /tmp/pylite_dbfarm
# Start processing logs sql/sql_logs version 52200
# Finished processing logs sql/sql_logs
{'i': array([    0,     1,     2, ..., 19997, 19998, 19999]), 'k': array([40000, 40001, 40002, ..., 59997, 59998, 59999]), 'j': array([20000, 20001, 20002, ..., 39997, 39998, 39999]), 'm': array([80000, 80001, 80002, ..., 99997, 99998, 99999]), 'l': array([60000, 60001, 60002, ..., 79997, 79998, 79999])}
Count 20000

# 14:17:07 >  
# 14:17:07 >  "Done."
# 14:17:07 >  

